src/screens/Customers/CustomerProfile.tsx

```diff
@@ -118,10 +157,16 @@ function CustomerProfile(props: CustomerProps) {
           Meter Reading
         </Typography>
         <OutlinedColCard info={readingInfo} primary={true} rightIcon={getAddButton()} />
-        <OutlinedColCard info={info3} primary={false} />
+        <OutlinedColCard info={meterInfo} primary={false} />
       </div>
     </div>
   );
 }
 
-export default withStyles(styles)(CustomerProfile);
+const mapStateToProps = (state: RootState) => {
+  let customers = state.siteData.currentSite.customers;
+  let siteName = state.siteData.currentSite.name;
```

Comment:
We should handle `currentSite` being potentially null just to be defensive (at the very least we shouldn't crash!). We can just set these to an empty list and empty string (or some placeholder like `Site Name`)